tests/runtime/call

```diff
@@ -278,6 +278,8 @@ ENV BPFTRACE_CACHE_USER_SYMBOLS=PER_PROGRAM
 PROG uprobe:./testprogs/uprobe_symres_exited_process:test { print(ustack); exit(); }
 EXPECT_REGEX ^\s+test\+[0-9]+\s+test2\+[0-9]+\s+main\+[0-9]+
 AFTER ./testprogs/disable_aslr ./testprogs/uprobe_symres_exited_process
+# Required to skip function prologue
+REQUIRES_FEATURE dwarf
```

Comment:
driveby fixes?

Reply:
Oh, that change merged already. Lemme rebase onto it